namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-preprod/01-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hmpps-contacts-preprod-admin
+  namespace: hmpps-contacts-preprod
+subjects:
+  - kind: Group
+    name: "github:dps-tech"
+    apiGroup: rbac.authorization.k8s.io
+  - kind: Group
+    name: "github:hmpps-move-and-improve"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
```

Comment:
The service reliability team (SRE) generally want to be added to RBAC for CP please 
```
- kind: Group
    name: "github:hmpps-sre"
    apiGroup: rbac.authorization.k8s.io
````